hypervisor/Documentation/ACRN_UEFI.txt

```diff
@@ -11,3 +11,10 @@ d. Update the bsp/uefi/clearlinux/acrn.conf file by filling the field <UUID of r
 e. Copy bsp/uefi/clearlinux/acrn.conf to loader\\entries\\ directory.
 f. Reboot.
 g. If you want to recover to original os loader, just replace the bootloaderx64.efi with bootloaderx64_origin.efi.
+
+Note:
```

Comment:
Thanks for taking the time to write these notes @JasonChenCJ 

This file will very shortly be deleted however as *all* documentation should be put under `doc/`. You could enhance [this section](https://github.com/projectacrn/acrn-hypervisor/blob/master/doc/primer/index.rst#memory-virtualization) in the Developer Primer or create a new tech note under the "How-To" section.